src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySink.java

```diff
@@ -231,7 +231,7 @@ private void configureBigQuerySink() {
    */
   private void configureTable(Schema schema) {
     AbstractBigQuerySinkConfig config = getConfig();
-    Table table = BigQueryUtil.getBigQueryTable(config.getProject(), config.getDataset(),
+    Table table = BigQueryUtil.getBigQueryTable(config.getDatasetProject(), config.getDataset(),
```

Comment:
what if `config.getProject()` is provided? In that case we will rely on `config.getDatasetProject()`?

Reply:
If `config.getProject()` and `config.getDatasetProject()` are provided, then `config.getDatasetProject()` will get priority. 

That's how it is defined in **AbstractBigQuerySinkConfig**
https://github.com/data-integrations/google-cloud/blob/86f4b91d9bf2916f48ba9fd12aacacecc8e805e5/src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java#L114-L120